web/src/components/SearchResultMatch.tsx

```diff
@@ -77,7 +77,7 @@ export class SearchResultMatch extends React.Component<SearchResultMatchProps, S
                                 }).pipe(
                                     switchMap(highlightedStr => {
                                         const highlightedMarkdown = decode(markdownHTML).replace(
-                                            codeContent,
+                                            '<pre>' + codeContent + '\n</pre>',
```

Comment:
@attfarhan

Reply:
As much as possible, we don't want to rely on the HTML structure of the rendered markdown. You actually just need to remove the `trim()` above so that the new line is also removed (see suggestion above). 
```suggestion
                                            codeContent,
```